app/assets/stylesheets/application.css.scss

```diff
@@ -360,6 +338,41 @@ font-size: 13px
   display: block;
 }
 
+.admin-dash{
+  font-family: 'Oswald', sans-serif;
+  h2{
+    color: #ffab40
+  }
+  a{
+    font-family: 'Oswald', sans-serif;
+    color: #ffab40
+  }
+  a.delete{
+    font-family: 'Oswald', sans-serif;
+    color: white
+  }
+  ul.collection {
+    background: white;
+  }
+  li {
+    padding-left: 15px;
+  }
+  .collection .collection-item.avatar {
+    padding-left: 150px;
+  }
+  .collection .collection-item.avatar .circle {
```

Comment:
Merge rule `.collection .collection-item.avatar .circle` with rule on line 360<br>Selector should have depth of applicability no greater than 2, but was 4